exercises/concept/calculator-conundrum/.docs/intoduction.md

```diff
@@ -0,0 +1,122 @@
+# Exception Handling in Java
+
+The Exception Handling in Java is one of the powerful mechanism to handle the runtime errors so that the normal flow of the application can be maintained.
+
+Good exception handling can handle errors and gracefully re-route the program to give the user still a positive experience.
```

Comment:
To avoid repetition:

Good exception handling can re-route the program to maintain a positive user experience.

---

Also, I would put this in the same paragraph with the first sentence (so no intervening empty line).